src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -1599,7 +1600,8 @@ void java_lang_Thread::clear_scopedValueBindings(oop java_thread) {
 }
 
 oop java_lang_Thread::holder(oop java_thread) {
-    return java_thread->obj_field(_holder_offset);
+  // Note: may be null if the thread is still attaching
```

Comment:
nit perhaps: s/may be null/may return null/

Reply:
Changed